_data/working_groups/frontend.yml

```diff
@@ -1,6 +1,17 @@
 ---
 name: Frontend
 full_name: Frontend Guild
+mission: |
```

Comment:
Hey @meiqimichelle, I'm pretty sure this should all go in the data-private version.